package.json

```diff
@@ -3,25 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.0",
+    "@reduxjs/toolkit": "^1.6.2",
```

Comment:
- [ ] Kindly update the readme following [this template](https://github.com/microverseinc/readme-template).

Reply:
The README is now professionally documented. ✔️